parts/plugins/x-php.py

```diff
@@ -59,6 +59,7 @@ def schema(cls):
         return schema
 
     def __init__(self, name, options, project):
+        options.configflags.extend(['--with-libdir=lib/{}'.format(project.arch_triplet)])
```

Comment:
Good point. You need this when enabling some features which ship with PHP, but it might also be needed for extensions.
What the best way to do this? Simply duplicate the line?
Thinking about lib64, I think it didn't work and that's the reason I had to try something else, but before I try again. How would we get the proper path based on the arch since lib32 doesn't exist on armhf?